docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+version: '3'
+
+services:
+  php:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    restart: unless-stopped
+    volumes:
+      - ./:/usr/share/nginx/html
+
+  nginx:
+    image: nginx:alpine
```

Comment:
Thoughts on pulling this out into its own docker image? `.docker/nginx/Dockerfile` where the docker file could contain a `COPY` command with the conf and public files

Reply:
The advantage of mounting conf files is that the container image doesn't need to be rebuilt every time a config file is changed.